free-2.md

```diff
@@ -58,12 +94,16 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+Almost 99% of the work would be on the system looking up recipes to recommend. 
 
 ### Scope
 _How big an idea is this? How ambitious is this project?_
 
+This project because of the web scraping would be ambitious for the semester. 
 
 ### Benefits and drawbacks
 _Why might this be a good idea for a project? Why might this not be a good idea 
 project?_
 
+This would be a bad idea since it doesn't allow for that much user expressiveness and
+the backend interpreter for the language would require difficult web scraping. 
```

Comment:
Again, an interesting project. But your current description focuses more on infrastructure than language design. If 99% of the time is spent on the system for looking up recipes, then it probably doesn't make a good DSLs project. There are ways to have a DSLs project for this domain, though, if you're interested. And a few other people in the class have proposed ideas related to recipes, which might give you some inspiration for a language in this domain.